### DISCUSSION TITLE
ESlint `key-spacing` equivalent?

### DISCUSSION THREAD

**Discussion by codekoala** [biomejs/biome#3215] [Q&A]
I've searched using as many keywords as I can manage, but I haven't been able to find a way to tell biome how to do something similar to eslint's `key-spacing` (https://eslint.org/docs/latest/rules/key-spacing). Is this currently supported in any fashion?

**Comment by ematipico** [comment]
Please, use this discussion to propose other rules: https://github.com/biomejs/biome/discussions/3

FYI we won't support/accept stylistic rules
